js/main.js

```diff
@@ -53,6 +53,16 @@ $(".search-close").on("click", function() {
   $('body').removeClass('search-on');
 });
 
+$("#search-input").on("input", function() {
+  var content = $(this).val();
+  $('#search-input').css({'background-image' : '', 'background-repeat' : '', 'background-size':'', 'background-position': ''});  
+  if(content=='')
+  {
+    $('#search-input').css({'background-image' : 'url(../images/icon-search.svg)', 'background-repeat' : 'no-repeat', 'background-size':'18px 18 px', 'background-position': 'right 14px center'});
```

Comment:
Please avoid inline styles as it leads to code redundancy. Try playing around with opacity or jquery attribute fetch